### PR TITLE
testing/perl-text-unidecode: new aport

### PR DESCRIPTION
**Pull request by timlegge**

### CODE REVIEW

File: testing/perl-text-unidecode/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-text-unidecode
+_pkgreal=Text-Unidecode
+pkgver=1.30
+pkgrel=0
+pkgdesc="Provide plain ASCII transliterations of Unicode text"
+url="https://metacpan.org/release/Text-Unidecode/"
+arch="noarch"
+license="GPL-1.0-or-later Artistic-1.0-Perl"
```
Comment (maxice8) [inline testing/perl-text-unidecode/APKBUILD]: use 'OR' SPDX identifier to indicate the choice is between either of those licenses.

Comment (timlegge) [reply testing/perl-text-unidecode/APKBUILD]: @maxice8 I made the change.  Wondered about that

Comment (maxice8) [reply testing/perl-text-unidecode/APKBUILD]: Thank you

Comment (timlegge) [reply testing/perl-text-unidecode/APKBUILD]: also fixed apkbuild-cpan.in PR tto abuild

**Comment by maxice8** [comment]
On Thu, Jun 06, 2019 at 06:03:35PM -0700, Timothy Legge wrote:
> timlegge commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,33 @@
> +# Automatically generated by apkbuild-cpan, template 2
> +# Contributor: Timothy Legge <timlegge@gmail.com>
> +# Maintainer: Timothy Legge <timlegge@gmail.com>
> +pkgname=perl-text-unidecode
> +_pkgreal=Text-Unidecode
> +pkgver=1.30
> +pkgrel=0
> +pkgdesc="Provide plain ASCII transliterations of Unicode text"
> +url="https://metacpan.org/release/Text-Unidecode/"
> +arch="noarch"
> +license="GPL-1.0-or-later Artistic-1.0-Perl"
> 
> also fixed apkbuild-cpan.in PR tto abuild
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8556#discussion_r291423494

Nice

**Pull request closed without merging**